scripts/count-median.py

```diff
@@ -91,7 +91,10 @@ def main():
     ksize = htable.ksize()
 
     print('writing to', output_filename, file=sys.stderr)
-    output = open(output_filename, 'w')
+    if output_filename in ('-', '/dev/stdout'):
+        output = sys.stdout
```

Comment:
Shouldn't the output argument on line 63 be set to argparse.FileType instead of this?